package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "json-python-coverage",
```

Comment:
Name should include the org namespace (in this case `@cz-nic/json-python-coverage`).

Reply:
Fixed